src/containers/DeveloperPortalProjects/components/Breadcrumb/index.tsx

```diff
@@ -15,11 +15,17 @@ import './Breadcrumb.scss';
 type Props = {
   className?: string;
   breadcrumbHeight: number;
+  approvedProjectUrls?:
+    | {
+        title: string;
+        url: string;
+      }[]
+    | null;
```

Comment:
😮 What's this? It can be `null` or an array of objects with the shape `{title, url}`?

What does that first pipe mean?

Reply:
> 😮 What's this? It can be null or an array of objects with the shape {title, url}?

Yes correct.

The first pipe is needed if your type starts in the next line, autoformat does that 😅